src/widget/weffectparameterbase.cpp

```diff
@@ -21,7 +21,11 @@ void WEffectParameterBase::setEffectParameterSlot(
 
 void WEffectParameterBase::parameterUpdated() {
     if (m_pEffectParameterSlot) {
-        setText(m_pEffectParameterSlot->name());
+        if (! m_pEffectParameterSlot->shortName().isEmpty()) {
+            setText(m_pEffectParameterSlot->shortName());
+        } else {
+            setText(m_pEffectParameterSlot->name());
+        }
```

Comment:
I think it'd be nice to setBaseTooltip as something like ```<name>: <description>``` so it's easy to learn the full name. (same in WEffect)